cmd/argo/lint/lint.go

```diff
@@ -78,12 +86,45 @@ func GetFormatter(fmtr string) (Formatter, error) {
 	return f, nil
 }
 
+// RunLint lints the specified kinds in the specified files and prints the results to os.Stdout.
+// If linting fails it will exit with status code 1.
+func RunLint(ctx context.Context, client apiclient.Client, files, kinds []string, defaultNs, output string, strict bool) {
+	fmtr, err := GetFormatter(output)
+	errors.CheckError(err)
```

Comment:
I think if this is to work in third party software, it needs to return err rather than panic.

Reply:
Actually I meant to keep `RunLint` as a function to be called our CLI, and `Lint` can be called by other libraries to analyze the lint results and do whatever they want with them.